frontend/src/lib/components/TaxonomicFilter/InfiniteList.tsx

```diff
@@ -268,6 +268,7 @@ export function InfiniteList(): JSX.Element {
         totalResultCount,
         totalListCount,
         expandedCount,
+        showPopper,
```

Comment:
Perhaps `popper` -> `popover` ?

"Popper" is just the name of the library, but "popover" (or "popup"?) is what we want to show.